lib/logstash/inputs/udp.rb

```diff
@@ -97,10 +101,12 @@ def inputworker(number)
           decorate(event)
           event.set("host", client[3]) if event.get("host").nil?
           @output_queue.push(event)
+          metric.increment(:events)
```

Comment:
If we want this done in the codec, and I agree with doing that, I think we have two options for this PR to move forward:

1. Leave as-is, and later refactor once codecs are capable of counting events
2. Remove event counts from this PR.